issues.md

```diff
@@ -171,60 +165,41 @@ We need an assigned triager to regularly triage bugs coming in on each MDN conte
+  - Label: `p0` (will be addressed immediately)
 
-Definitions:
+- Major issue: This type of issue could severely affect a page's usefulness. For example, a significant amount of out-of-date information, a complex and important code example that doesn't work, a significant amount of prose that is badly written and hard to understand, or a large number of broken links.
+  - Labels: `p1` (will be addressed soon) and `p2` (will be addressed soon, but higher priority items will take precedence)
 
-- Critical issue — Something that could damage MDN's reputation severely and/or harm users, which we need to fix as soon as possible, regardless of where it appears on the site. Examples include code examples that if used in production could create a severe security issue, undesirable content such as malware, profanity, pornography, hate speech, or other undesirable content, or links to such content.
-- Major issue — Something that could severely affect a page's usefulness, for example a significant amount of out-of-date information, a complex and important code example that doesn't work, a significant amount of prose that is badly written and hard to understand, a large number of broken links, etc.
-- Minor issue — something that doesn't look great but does not affect learning, or only has a minor effect on learning. Examples — Typos, bad grammar, a broken link, a small amount of out-of-date information or badly-written prose, a small code snippet that doesn't work.
+- Minor issue — This is a type of improvement issue that can make the existing content better but does not affect learning or only has a minor effect on learning. Examples include typos, bad grammar, a broken link, a small amount of out-of-date information or badly-written prose, or a code snippet that doesn't work.
+  - Labels: `p3` (no visibility when the issue will be addressed)
 
-Generally speaking, critical issues should be fixed immediately, and would probably be handled by MDN staff/peers.
+In general, critical issues should be fixed immediately and are most likely handled by MDN staff and peers.
 
-### Provide further information
+#### Add helpful information
 
-It is really useful for other contributors to provide them with further information to help them fix issues; we'd like to recommend that triaging each bug involves a time-box of up to 5 minutes in which the triager quickly describes some steps to take to fix the bug, to help the person who eventually tries to fix it.
+If possible, add information that can help contributors to fix the issue. The information can be in the form of steps, direction, or reading resources. You can time-box this task to 5 minutes.
```

Comment:
We should call out that this is especially important for `good first issues`.

Reply:
Done in https://github.com/mdn/temp-processes/pull/7/commits/2a42225e4e69ce3ebf6e7b6aa361110458d97e59